### PR TITLE
[DOCS] Consolidates KSPM FAQs into one page

### LINKED ISSUE #2622: [DOCS] Consolidate KSPM FAQ sections
## Description 

The pages within the KSPM docs section have FAQs at the bottom. These should be consolidated into a new page in the KSPM section called FAQ (removed from their current locations).

## Acceptance Test Criteria

Reviewed by security docs team

### PR DESCRIPTION
**Pull request by benironside**
fixes #2622 

Consolidates the FAQs from all the KSPM pages to a single page. 

Tinsae, please double-check for the accuracy of answers, and feel free to add any other FAQs.

Everyone, please share your thoughts on how the FAQ are organized. Do the sections seem helpful and intuitive?

Preview: [KSPM FAQ](https://security-docs_2625.docs-preview.app.elstc.co/guide/en/security/master/kspm-faq.html)

### CODE REVIEW

File: docs/cloud-native-security/kspm-faq.asciidoc [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Frequently asked questions related to the Kubernetes Security Posture Management (KSPM) integration.
+
+[discrete]
+== General KSPM FAQ
+
+*Can I evaluate the security posture of my Amazon EKS clusters?*
+Yes. KSPM currently supports the security posture evaluation of Amazon EKS and unmanaged Kubernetes clusters.
+
+*How often is my cluster’s security posture evaluated?*
+Clusters are evaluated upon deployment of a KSPM integration, and every four hours after that.
```
Comment (nastasha-solomon) [inline docs/cloud-native-security/kspm-faq.asciidoc:12]: What do you think about combining the "General KSPM FAQ" section with the Cluster enrollment FAQ" section? Both sections seem to address common questions about clusters, so maybe you could group them together under a common section titled "Cluster enrollment and security", "Cluster monitoring", or just "Clusters".

File: docs/cloud-native-security/kspm-faq.asciidoc [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+[discrete]
+=== Findings page FAQ
+
+*Are all the findings on the Findings page current?*
```
Comment (nastasha-solomon) [inline docs/cloud-native-security/kspm-faq.asciidoc:17]:
```suggestion
*Are all the findings current?*
```

File: docs/cloud-native-security/kspm-faq.asciidoc [hunk 3]
```diff
@@ -0,0 +1,51 @@
+[[kspm-faq]]
+== KSPM FAQ
+Frequently asked questions related to the Kubernetes Security Posture Management (KSPM) integration.
```
Comment (nastasha-solomon) [inline docs/cloud-native-security/kspm-faq.asciidoc:3]:
```suggestion
Frequently asked questions about the Kubernetes Security Posture Management (KSPM) integration and features.
```

File: docs/cloud-native-security/kspm-faq.asciidoc [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+Clusters are evaluated upon deployment of a KSPM integration, and every four hours after that.
+
+[discrete]
+=== Findings page FAQ
+
+*Are all the findings on the Findings page current?*
+Yes. Only the most recent findings appear on the Findings page.
+
+*Can I build custom visualizations and dashboards that incorporate findings data?*
+Yes. You can use {kib}'s custom visualization capabilities with findings data. Current findings data can be accessed using the `logs-cloud_security_posture.findings_latest-*` index pattern, while historical findings data can be accessed using the `logs-cloud_security_posture.findings-*` index pattern. To learn more, refer to {kibana-ref}/dashboard.html[Dashboards and visualizations].
```
Comment (nastasha-solomon) [inline docs/cloud-native-security/kspm-faq.asciidoc:21]: Do you need the second half about where to find current and historical data? Or is that information more relevant to the next question? 

```suggestion
Yes. You can use findings data to build {kibana-ref}/dashboard.html[custom visualizations] in {kib}. 
```

File: docs/cloud-native-security/kspm-faq.asciidoc [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+Resources are fetched and evaluated against benchmark rules when a KSPM integration is deployed, and every four hours after that.
+
+*Can I configure the KSPM integration's fetch cycle?*
+No, the four-hour fetch cycle is not configurable.
+
+*Can I contribute to the CSP ruleset?*
+You can't directly edit benchmark rules. The rules are defined https://github.com/elastic/csp-security-policies[in this repository], where you can raise issues with certain rules. They are written in https://www.openpolicyagent.org/docs/latest/policy-language/[Rego].
+
+*How can I tell which specific version of the CIS benchmarks is in use?*
+Refer to the `rule.benchmark.name` and `rule.benchmark.version` fields for documents in these datastreams: `logs-cloud_security_posture.findings-default` and `logs-cloud_security_posture.findings_latest-default`
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/cloud-native-security/kspm-faq.asciidoc:42]: Made this suggestion to visually break up the last part of the answer which looks very stylistically similar.

**Review by nastasha-solomon — COMMENTED** [review]
Looks really great and should be a useful page! I did have a couple of general suggestions about the page order and structure/design:

- **Page order:** It could be helpful to order the sections to match the feature's overall workflow. This would teach new users what the general flow is, and enforce the mental model for more experienced users. Here's my understanding of what that flow looks like: 1.) Install the integration 2.) Enroll clusters 3.) Set up benchmark rules 4.) Review findings
  - If the order is correct, I'd suggest ordering the sections as so:
    * Cluster enrollment and security
    * Benchmark rules
    * Findings
- **Structure/design:** How do you feel about creating collapsible sections for each question, similar to the way the [Detection rules](https://www.elastic.co/guide/en/security/master/ts-detection-rules.html) topic is set up. Right now, there's some visual distinction between the question and the answer, but I think adding more could help improve the page's readability and usability. 


Hopefully these are helpful - great job overall!


Edit: I did have one more question. Should the section titles be L2? Right now, they seem to be a sub-section under the "General KSPM FAQ" section.

File: docs/cloud-native-security/kspm-faq.asciidoc [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+== General KSPM FAQ
+
+*Can I evaluate the security posture of my Amazon EKS clusters?*
+Yes. KSPM currently supports the security posture evaluation of Amazon EKS and unmanaged Kubernetes clusters.
+
+*How often is my cluster’s security posture evaluated?*
+Clusters are evaluated upon deployment of a KSPM integration, and every four hours after that.
+
+[discrete]
+=== Findings page FAQ
```
[the remark this answers is not in the document]
Comment (joepeeples) [reply docs/cloud-native-security/kspm-faq.asciidoc:15]: +++

File: docs/cloud-native-security/kspm-faq.asciidoc (hunk 1 above)
Comment (joepeeples) [inline docs/cloud-native-security/kspm-faq.asciidoc:12]: +1 on combining these sections per @nastasha-solomon. For a relatively short page, it probably doesn't need to be so heavily segmented and granular.

Also a possible edit for tone:
```suggestion
Clusters are evaluated when you deploy a KSPM integration, and every four hours after that.
```

File: docs/cloud-native-security/kspm-faq.asciidoc [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+*Are all the findings on the Findings page current?*
+Yes. Only the most recent findings appear on the Findings page.
+
+*Can I build custom visualizations and dashboards that incorporate findings data?*
+Yes. You can use {kib}'s custom visualization capabilities with findings data. Current findings data can be accessed using the `logs-cloud_security_posture.findings_latest-*` index pattern, while historical findings data can be accessed using the `logs-cloud_security_posture.findings-*` index pattern. To learn more, refer to {kibana-ref}/dashboard.html[Dashboards and visualizations].
+
+*Can I view past findings?*
+Yes. Historical findings data is accessible using the `logs-cloud_security_posture.findings-*` index pattern.
```
[the remark this answers is not in the document]
Comment (joepeeples) [reply docs/cloud-native-security/kspm-faq.asciidoc:24]: ❤️❤️❤️ the idea of a list for this kind of info. Easily scannable, neat and tidy.

Also, it seems like this FAQ and the previous one could maybe be combined, or each one revised to be more focused -- that looks like what @nastasha-solomon is getting at in the previous comment. They're both answering the same question, how do I find and visualize current and past findings?

File: docs/cloud-native-security/kspm-faq.asciidoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (joepeeples) [reply docs/cloud-native-security/kspm-faq.asciidoc:42]: ++

**Review by joepeeples — COMMENTED** [review]
Consolidated FAQ page looks great!
